src/atomics/fences.md

```diff
@@ -0,0 +1,257 @@
+# Fences
+
+As well as loads, stores, and RMWs, there is one more kind of atomic operation
+to be aware of: fences. Fences can be triggered by the
+[`core::sync::atomic::fence`] function, which accepts a single ordering
+parameter and returns nothing. They don’t do anything on their own, but can be
+thought of as events that strengthen the ordering of nearby atomic operations.
+
+## Acquire fences
+
+The most common kind of fence is an _acquire fence_, which can be triggered in
+three different ways:
+1. `atomic::fence(atomic::Ordering::Acquire)`
+1. `atomic::fence(atomic::Ordering::AcqRel)`
+1. `atomic::fence(atomic::Ordering::SeqCst)`
+
+An acquire fence retroactively makes every single non-`Acquire` operation that
```

Comment:
This is subtly wrong. fence will not turn relaxed store into an Acquire operation. Also, fence will not mirucalously turn non-atomic read to Acquire load. I think it would be better to re-phrase this.
I also found "in other words, it causes every prior `Release`d value that was" hard to comprehend. 
I tried to come up with a better wording but I am not sure it is 100% clear yet:
"in other words, for any (even relaxed) atomic load sequenced before the fence, if the load reads a value that was `Released`d by some operation O, it creates synchronizes-with relationship between the fence and O.